packages/date/src/index.js

```diff
@@ -75,56 +85,20 @@ let settings = {
 		},
 	},
 	formats: {
-		time: 'g: i a',
+		time: 'g:i a',
 		date: 'F j, Y',
 		datetime: 'F j, Y g: i a',
 		datetimeAbbreviated: 'M j, Y g: i a',
 	},
 	timezone: { offset: '0', string: '', abbr: '' },
 };
-
 /**
  * Adds a locale to moment, using the format supplied by `wp_localize_script()`.
  *
  * @param {Object} dateSettings Settings, including locale data.
  */
 export function setSettings( dateSettings ) {
 	settings = dateSettings;
```

Comment:
It might be necessary to set all those settings directly to `moment` script handle that some plugins still use. See https://github.com/WordPress/gutenberg/issues/27159 that reports error on `tz` property.

Reply:
I have more details included in https://github.com/WordPress/gutenberg/issues/27159#issuecomment-731651670.

Reply:
> I have more details included in [#27159 (comment)](https://github.com/WordPress/gutenberg/issues/27159#issuecomment-731651670).

Thanks for taking a look, I'll do the same and reply on the issue you linked.